packages/preact-query-devtools/vite.config.ts

```
@@ -0,0 +1,27 @@
+import preact from '@preact/preset-vite'
+import { defineConfig } from 'vitest/config'
+import type { UserConfig as ViteUserConfig } from 'vite'
+import packageJson from './package.json'
```

ESLint

[error] `./package.json` import should occur before type import of `vite`

(import/order)